src/puppetlabs/puppetdb/metrics/core.clj

```diff
@@ -7,7 +7,8 @@
     {:registry registry
      :reporter (reporter registry {:domain domain})}))
 
-(def metrics-registries {:mq (new-metrics "puppetlabs.puppetdb.mq")
+(def metrics-registries {:clean (new-metrics "puppetlabs.puppetdb.clean")
```

Comment:
I might be tempted to put these metrics under `puppetlabs.puppetdb.admin` instead